pkg/scheduler/profile/profile.go

```diff
@@ -99,6 +99,9 @@ func (v *cfgValidator) validate(cfg config.KubeSchedulerProfile) error {
 	if len(cfg.Plugins.QueueSort.Enabled) != 1 {
```

Comment:
the old check would return separate errors for `len == 0` and `len > 1`, could we copy that here?